shepherd/Utils.py

```diff
@@ -33,13 +33,18 @@ class SCOREBOARD_HEADER(Enum):
     POWERUPS = "powerups"
     ALLIANCE_MULTIPLIER = "alliance_multiplier"
 
-@unique # pylint: disable=invalid-name
+# pylint: disable=invalid-name
 class CONSTANTS(Enum):
-    TWO_X_COOLDOWN = "two_x_cooldown"
-    ZERO_X_COOLDOWN = "zero_x_cooldown"
-    STEAL_COOLDOWN = "steal_cooldown"
-    CODE_COOLDOWN = "code_cooldown"
-    BID_INCREASE_CONSTANT = "bid_increase_constant"
+    TWO_X_COOLDOWN = 5
+    ZERO_X_COOLDOWN = 5
+    STEAL_COOLDOWN = 5
+    CODE_COOLDOWN = 5
+    BID_INCREASE_CONSTANT = 5
+    GOLD = "gold"
```

Comment:
As mentioned before, I think `CONSTANTS` should just be for things with numerical values.